tests/test_self_update.cpp

```
@@ -226,6 +232,86 @@ TEST_F(InstallStdlibTest, OldLayoutArchiveInstallsToLibStd) {
+    bool ok = install_stdlib(src_dir.string(), "0.1.0");
+    EXPECT_TRUE(ok);
+}
+
+class InstallNativeLibsTest : public RyInstallTestBase {
+protected:
+    void SetUp() override {
+        SetUpDirs("ry_test_install_native_libs");
+    }
+};
+
+TEST_F(InstallNativeLibsTest, ReturnsFalseWhenNoLibDirectory) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)